README.md

```diff
@@ -170,6 +170,35 @@ class PostSerializer < ActiveModel::Serializer
 end
 ```
 
+### Overriding the Key Format
```

Comment:
:+1: It would be awesome if along with this PR you could add some Docs to our [new documentation](https://github.com/rails-api/active_model_serializers/tree/master/docs) we have being working on before release.
Maybe event add some article about integrating it with ember.js using our `How to` section.